pkg/controller_workload/workload/workload_event.go

```diff
@@ -514,3 +521,25 @@ func handleAddressTypeResponse(rsp *service_discovery_v3.DeltaDiscoveryResponse)
 	}
 	return err
 }
+
+func (svc *ServiceEvent) handleAuthorizationTypeResponse(rsp *service_discovery_v3.DeltaDiscoveryResponse) error {
+	auth := &security.Authorization{}
+
+	// update resource
+	for _, resource := range rsp.GetResources() {
+		if err := anypb.UnmarshalTo(resource.Resource, auth, proto.UnmarshalOptions{}); err != nil {
+			log.Errorf("unmarshal failed, err: %v", err)
+			continue
+		}
+		log.Debugf("handle auth xds, resource.name %s, auth %s", resource.GetName(), auth.String())
+		_ = svc.rbac.UpdatePolicy(auth)
+	}
+
+	// delete resource by name
+	for _, rmResourceName := range rsp.GetRemovedResources() {
+		_ = svc.rbac.RemovePolicy(rmResourceName)
```

Comment:
If you get an error when deleting RBAC, do you need to return an error message?

Reply:
There is no error when deleting RBAC, so we ignore the return value here